### PR TITLE
963 dpl 707 as developers we need to create the smrt link 12 revio version options sample sheet headers so users can have the correct columns on the sample sheet

### PR DESCRIPTION
**Pull request by enegoktan**
Closes #

Changes proposed in this pull request:

*
*
* ...

### CODE REVIEW

File: config/pacbio_smrt_link_versions.yml [hunk 1]
```diff
@@ -149,6 +154,51 @@ default: &default
       versions:
         - v10
         - v11
```
Comment (BenTopping) [inline config/pacbio_smrt_link_versions.yml:156]: I would have a look to see if any of the existing values also apply to v12. https://www.pacb.com/wp-content/uploads/SMRT_Link_User_Guide_v12.0.pdf

File: config/pipelines/pacbio.yml [hunk 2]
```diff
@@ -423,6 +423,7 @@ default: &default
               for:
                 - :well
               with: :well
+      # v12
```
Comment (BenTopping) [inline config/pipelines/pacbio.yml:426]: assume this is going to be done in DPL-710

File: spec/models/pacbio/well_spec.rb [hunk 3]
```diff
@@ -317,5 +324,77 @@
...
+            expect(well).to be_valid
+          end
+
+          well.include_base_kinetics = 'junk'
+          expect(well).not_to be_valid
+        end
+      end
+
+      context 'Library concentration' do
+        it 'must have a library concentration' do
```
Comment (BenTopping) [inline spec/models/pacbio/well_spec.rb:386]: Would add a test here and in the 'polymerase kit' tests to check it is valid when it has a valid value

File: config/pipelines/pacbio.yml (hunk 2 above)
Comment (enegoktan) [reply config/pipelines/pacbio.yml:426]: yes, that's the sample sheet config story

File: config/pacbio_smrt_link_versions.yml (hunk 1 above)
Comment (enegoktan) [reply config/pacbio_smrt_link_versions.yml:156]: Just had another look, don't think there's any other fields that are applicable to v12_revio

**Review by BenTopping — APPROVED** [review]

**Pull request merged**